sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/_setup_scripts.py

```diff
@@ -47,7 +47,7 @@ def _from_rest_object(cls, obj: RestScriptReference) -> Optional["ScriptReferenc
         script_reference = ScriptReference(
             path=obj.script_data if obj.script_data else None,
             command=obj.script_arguments if obj.script_arguments else None,
-            timeout_minutes=timeout_minutes,
+            timeout_minutes=timeout_minutes,  # type: ignore[arg-type]
```

Comment:
If this method is looking for a str, you can always call `str(timeout_minutes)` to make the mypy error go away.

Reply:
this method is looking for `Optional[int]`, I fixed this line with `cast(Optional[int], timeout_minutes)`